openstef_dbc/services/write.py

```diff
@@ -247,6 +276,41 @@ def write_weather_data(
 
         return message
 
+    def write_realised_power_measurements(self, df: pd.DataFrame, sid: str):
+        """Method that writes measurement data to the influx database.
+
+        Args:
+            df: pd.DataFrame(index = "datetime", columns = ['output'])
+            sid: (str) String with system id of the measurement
+
+        Returns:
+            None
+        """
+        df["type"] = "measurement"
+        df["system"] = str(sid)
+        df["created"] = int(time.time())
+        df = df.astype({"output": np.float64})
+        # Write to influx database
+        result = _DataInterface.get_instance().exec_influx_write(
+            df,
+            database="realised",
+            measurement="power",
+            tag_columns=["system", "type"],
+            field_columns=["output", "created"],
+            time_precision="s",
+        )
+        if not result:
+            self.logger.error(
+                "Something wend wrong while writing measurement data to influx"
+            )
+            return
+
+        self.logger.info(
+            "Wrote measurement data for {} systems to influx".format(
+                df["system"].nunique()
+            )
+        )
+
```

Comment:
I see that the 'write_realised_pvdata' function also adds the system to mysql if it is a new system. De we want that here as well?

Reply:
No I dont think so as the writing has to be cleared via a valid api key. If such a key does not excist we do not want to make a mySQL entry (and also not write data to the influxDB).